CMakeLists.txt

```diff
@@ -1,17 +1,27 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(ign-rviz)
+add_compile_options(-std=c++14 -O2)
+set(PYTHON_EXECUTABLE "/usr/bin/python3")
```

Comment:
Please use the right Cmake functions to find python3

Reply:
Ok